src/components/new-request/search.vue

```diff
@@ -251,56 +251,60 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
-          if (this.showContinueInButton) return true
-          if (this.isFederal && this.isRestoration) return false
-          if (this.isFederal) return true
-
-          // don't show COLIN button for supported alteration entities
-          if (this.isConversion) {
-            return !this.isSupportedAlteration(this.getConversionType)
-          }
-
-          // don't show COLIN button for supported restoration entities
-          if (this.isRestoration) {
-            const supportedRestorationEntites = GetFeatureFlag('supported-restoration-entities')
-            const isRestorationEntity = supportedRestorationEntites.includes(this.entity_type_cd)
-            return !isRestorationEntity
-          }
-
-          if (this.isChangeName) {
-            const supportedChnageNameEntites = GetFeatureFlag('supported-name-change-entities')
-            const isChangeNameEntity = supportedChnageNameEntites.includes(this.entity_type_cd)
-            return !isChangeNameEntity
-          }
-
-          // don't show COLIN button for supported entities
-          const supportedEntites = GetFeatureFlag('supported-incorporation-registration-entities')
-          const isIncorporateEntity = supportedEntites.includes(this.entity_type_cd)
-          return !isIncorporateEntity
-        }
-      }
-    }
+    // *** TODO: add your logic here
+
+    // if (this.entity_type_cd || this.isFederal || this.isRestorable) {
+    //   if (this.isNumberedCompany || this.isFederal) {
+    //     if (!this.isConversion || this.isAlterOnline(this.getConversionType)) {
+    //       if (this.showContinueInButton) return true
+    //       if (this.isFederal && this.isRestoration) return false
+    //       if (this.isFederal) return true
+
+    //       // don't show COLIN button for supported alteration entities
+    //       if (this.isConversion) {
+    //         return !this.isSupportedAlteration(this.getConversionType)
+    //       }
+
+    //       // don't show COLIN button for supported restoration entities
+    //       if (this.isRestoration) {
+    //         const supportedRestorationEntites = GetFeatureFlag('supported-restoration-entities')
+    //         const isRestorationEntity = supportedRestorationEntites.includes(this.entity_type_cd)
+    //         return !isRestorationEntity
+    //       }
+
+    //       if (this.isChangeName) {
+    //         const supportedChnageNameEntites = GetFeatureFlag('supported-name-change-entities')
+    //         const isChangeNameEntity = supportedChnageNameEntites.includes(this.entity_type_cd)
+    //         return !isChangeNameEntity
+    //       }
+
+    //       // don't show COLIN button for supported entities
+    //       const supportedEntites = GetFeatureFlag('supported-incorporation-registration-entities')
+    //       const isIncorporateEntity = supportedEntites.includes(this.entity_type_cd)
+    //       return !isIncorporateEntity
+    //     }
+    //   }
+    // }
+
     return false
   }
 
```

Comment:
Interestingly, the logic for "Check this name" button, on line 332 below, seems to be correct!